client/signup/config/flows.js

```diff
@@ -374,13 +359,24 @@ const Flows = {
 			return flow;
 		}
 
-		if ( user.get() ) {
-			flow = removeUserStepFromFlow( flow );
+		// Replace design-type step with Store NUX flow steps if exists.
+		// Otherwise use Pressable store step for new English users as before
+		if ( includes( flow.steps, 'design-type' ) ) {
```

Comment:
Not sure if this is the best place, but we don't use the `signup/atomic-store-flow` feature flag anywhere here. If we wanted to turn the whole Atomic Store (`store-nux`) flow off, how would we do it?